notebooks/uncertainty_aware_models/labs/uncertainty_awareness_in_deep_learning.ipynb

```diff
@@ -98,16 +98,13 @@
    ]
```

Comment:
<p>Should add an instruction of how to create this kernel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done!